FWCore/Framework/interface/EventSetupRecordImpl.h

```diff
@@ -122,7 +122,7 @@ namespace edm {
       void clearProxies();
 
       ///Set the cache identifier and validity interval when starting a new IOV
```

Comment:
please update the comment to say what `hasFinder` does.

Reply:
I added a comment.